perfkitbenchmarker/vm_util.py

```diff
@@ -276,6 +277,57 @@ def RunThreaded(target, thread_params, max_concurrent_threads=200):
         '\n'.join([stacktrace for stacktrace in exceptions]))
 
 
+def _ExecuteProcCall(target_arg_tuple):
+  """Function invoked in another process by RunParallelProcesses.
+
+  Executes a specified function call.
+
+  Args:
+    target_arg_tuple: (target, args, kwargs) tuple containing the function to
+        call and the arguments to pass it.
+
+  Returns:
+    (result, exception) tuple. The first element is the return value from the
+    called function, or None if the function raised an exception. The second
+    element is the raised exception, or None if the function succeeded.
+  """
+  target, args, kwargs = target_arg_tuple
+  try:
+    return target(*args, **kwargs), None
+  except:
+    return None, traceback.format_exc()
+
+
+def RunParallelProcesses(target_arg_tuples, max_concurrency=None):
+  """Executes function calls concurrently in separate processes.
+
+  Args:
+    target_arg_tuples: list of (target, args, kwargs) tuples. Each tuple
```

Comment:
The difference here is that it is easy to execute different functions concurrently without needing to define a helper function that forwards to the real target. If we want this to be consistent with RunThreaded, I'd prefer to enhance RunThreaded to also support different targets, since we've already encountered PRs that could benefit from it.